packages/cashier/src/pages/p2p-cashier/p2p-cashier.tsx

```diff
@@ -154,6 +155,7 @@ const P2PCashier = observer(({ history, location }: TP2PCashierProps) => {
             setOrderId={setQueryOrder}
             setOnRemount={setOnRemount}
             setP2POrderProps={setP2POrderProps}
+            setP2PRedirectTo={setP2PRedirectTo}
```

Comment:
@ameerul-deriv I believe `@deriv/p2p` already can access `NotificationStore` directly, Do we still need to pass the method as a prop from `@deriv/cashier`? 🤔

Reply:
in the current p2p code base, we havent implemented the changes of getting the functions from cashier, for now I need to pass the function from cashier to p2p in order to redirect to the my profile page... What I can do for now is add a TODO to eventually remove this and directly access it from notification store from p2p in the card where I am removing p2p from cashier

Reply:
@ameerul-deriv We can ignore it for now 🙇🏻